zanata-client-commands/src/test/java/org/zanata/client/commands/init/SourceConfigPromptTest.java

```diff
@@ -109,4 +109,45 @@ public void allowUserToRetry() throws Exception {
                 Matchers.contains("*.*"));
         assertThat(opts.getExcludes(), Matchers.contains("a.pot"));
     }
+
+    @Test
+    public void canHandleFileProjectType() throws Exception {
+        when(fileClient.acceptedFileTypes()).thenReturn(ImmutableList.copyOf(DocumentType.values()));
+        // here we use absolute path because we create temp files in there
+        String expectedSrcDir = tempFolder.getRoot().getAbsolutePath() + "/resources";
+        ConsoleInteractor console =
+                MockConsoleInteractor.predefineAnswers(
+                        expectedSrcDir, "messages.txt",
+                        "*Excluded.txt", "PLAIN_TEXT[txt]", "y");
```

Comment:
Default extension for `PLAIN_TEXT` is already `[txt]`, so it would be better to use a different combination that is not default to prove that it is using both components.